### PR TITLE
Yubico version 5.0.5

### PR DESCRIPTION
**Pull request by DanPrs**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15876)

### CODE REVIEW

File: manifests/y/Yubico/Authenticator/5.0.5/Yubico.Authenticator.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Yubico.YubicoAuthenticator
```
Comment (KaranKad) [inline manifests/y/Yubico/Authenticator/5.0.5/Yubico.Authenticator.yaml:3]: PackageIdentifier: Yubico.Authenticator

**Review by KaranKad — CHANGES_REQUESTED** [review]

**Comment by KaranKad** [comment]
Oh! my review didn't really cause any changes you need to change PackageIdentifier in this file edit this file manifests/y/Yubico/Authenticator/5.0.5/Yubico.YubicoAuthenticator.locale.en-US.yaml

**Comment by DanPrs** [comment]
Re-ran `wingetcreate`
Hope that solves the pending issues.
Sorry about the trouble!

**Comment by KaranKad** [comment]
So much information is dropped from previous version.

**Comment by KaranKad** [comment]
Use this for following
`Yubico.Authenticator.installer.yaml`
```
PackageIdentifier: Yubico.Authenticator
PackageVersion: 5.0.5
MinimumOSVersion: 10.0.0.0
InstallModes:
  - interactive
  - silent
  - silentWithProgress
Installers:
  - Architecture: x64
    InstallerType: msi
    InstallerUrl: https://developers.yubico.com/yubioath-desktop/Releases/yubioath-desktop-5.0.5-win64.msi
    InstallerSha256: 580C825A5AF14989056144B90706E01070BF09F81EAABD0BE10BCEE2508EB2B9
    InstallerSwitches:
      Silent: /qn /norestart
      SilentWithProgress: /qb /norestart
    ProductCode: "{C4938DCC-540E-43DB-BDC2-962A94308617}"
#    Scope: 
    InstallerLocale: en-US
    UpgradeBehavior: install
  - Architecture: x86
    InstallerType: msi
    InstallerUrl: https://developers.yubico.com/yubioath-desktop/Releases/yubioath-desktop-5.0.5b-win32.msi
    InstallerSha256: 789E873C9B96C5F10A315227BF23FD68B3EA62C517BD0A03A80F6DDB4413FA72
    InstallerSwitches:
      Silent: /qn /norestart
      SilentWithProgress: /qb /norestart
    ProductCode: "{E736B756-59E5-4F93-A6B3-9B83A3011AB2}"
#    Scope: 
    InstallerLocale: en-US
    UpgradeBehavior: install
ManifestType: installer
ManifestVersion: 1.0.0
```
`Yubico.Authenticator.locale.en-US.yaml`
```
PackageIdentifier: Yubico.Authenticator
PackageVersion: 5.0.5
PackageLocale: en-US
Publisher: Yubico
PublisherUrl: https://www.yubico.com/
PublisherSupportUrl: https://support.yubico.com/hc/en-us
PrivacyUrl: https://www.yubico.com/support/terms-conditions/privacy-notice/
#Author: 
PackageName: Yubico Authenticator
PackageUrl: https://www.yubico.com/products/yubico-authenticator/
License: BSD-2-Clause License
LicenseUrl: https://raw.githubusercontent.com/Yubico/yubioath-desktop/master/COPYING
#Copyright: 
#CopyrightUrl: 
ShortDescription: Authenticator to generate 2-step verification codes using your YubiKey
#Description: 
Moniker: yubioath
Tags:
  - 2fa
  - security
  - auth
  - totp
ManifestType: defaultLocale
ManifestVersion: 1.0.0
```
`Yubico.Authenticator.yaml`
```
PackageIdentifier: Yubico.Authenticator
PackageVersion: 5.0.5
DefaultLocale: en-US
ManifestType: version
ManifestVersion: 1.0.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Please use `wingetcreate update` for existing packages so we don't loose the metadata.
Or well you can't on this since its a multi installer manifest, you could then use YamlCreate.ps1 instead tho. Or just manual update.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**